.husky/_/.gitignore

```diff
@@ -1 +1 @@
-_
\ No newline at end of file
+*
```

Comment:
what does this change do?

Reply:
To be honest, I don't know how that star was added because I have never opened this file before!

Reply:
But I have removed it

Reply:
got it 👍